modules/installation-minimum-resource-requirements.adoc

```diff
@@ -213,7 +213,7 @@ endif::ibm-power[]
 ifdef::azure[]
 [IMPORTANT]
 ====
-You are required to use Azure virtual machines with `premiumIO` set to `true`. The machines must also have the `hyperVGeneration` property contain `V1`.
+You are required to use Azure virtual machines with `premiumIO` set to `true`.
```

Comment:
Where do you set `premiumIO`? I can't tell from the context. I think this might be clearer if you rephrased it to something like this, but ideally it would have more info about where you find this setting. (This suggestion is phrased like it's a YAML field, so please take it or leave it.)

```suggestion
If you use Azure virtual machines, you must set the value of the `premiumIO` field to `true`.
```

Reply:
PS. You might already know this, but please don't click "Commit suggestion" :)

Reply:
This property PremiumIO is 'Premium disk support', this is the property of the Azure VM,  Some VM instance types, which PremiumIO is false, cannot be used in installer.